src/bases.jl

```diff
@@ -728,45 +724,49 @@ function gram_schmidt(
     M::AbstractManifold,
     p,
     V::AbstractVector;
+    atol = 5 * 1e-16,
```

Comment:
Could we replace this with something like `eps(number_eltype(V[1]))`? So that it also works for `Float32` for example.

Reply:
Sure. I just missed to do it more generic in the first place. The most recent commit solves this.